makefile

```diff
@@ -94,6 +94,7 @@ COMPILER_FLAGS =
 LINKER_FLAGS =
 
 
-## include the makefile-engine
-include /boot/develop/etc/makefile-engine
-
+## Include the Makefile-Engine
+DEVEL_DIRECTORY := \
+	$(shell findpaths -r "makefile_engine" B_FIND_PATH_DEVELOP_DIRECTORY)
+include $(DEVEL_DIRECTORY)/etc/makefile-engine
```

Comment:
Github lets me only comment on lines that were changed in the PR. Being lazy, I just clicked on a changed line and specified the lines I had comments on in the comment...